iron-iconset-svg.html

```diff
@@ -136,6 +136,10 @@
      */
     _nameChanged: function() {
       new Polymer.IronMeta({type: 'iconset', key: this.name, value: this});
+      this.async(function() {
```

Comment:
`iron-iconset` needs to run _mapThemes. Looks like iron-iconset-svg may not need the async though? Will need to investigate. I started with `iron-iconset` as it's what iron-icon's tests use (ignore the fact that I sent out the code reviews in the other order :cactus:).